File: week6/Abstract.cpp
#include<iostream>
using namespace std;

class AccessSpecifierDemo{
    private :
        int priVar;
    protected :
        int proVar;
    public :
        int pubVar;
    public :
        void setVar(int priValue,int proValue, int pubValue)
        {
            this->priVar = priValue;
            this->proVar = proValue;
            this->pubVar = pubValue;

        }
        void getVar(){

            cout << "Private variable: " << priVar << endl;
            cout << "Protected variable: " << proVar << endl;
            cout << "Public variable: " << pubVar << endl;

        }

};
int main()
{
    AccessSpecifierDemo obj;
    int a,b,c;
    cout << "Enter the value of private variable :";
    cin >> a;
    cout << "Enter the value of protected variable :";
    cin >> b;
    cout << "Enter the value of public variable :";
    cin >> c;
    obj.setVar(a,b,c);
    obj.getVar();

}

File: week6/Abstract.java
import java.util.Scanner;

class AccessSpecifierDemo{
    private int priVar;
    protected int proVar;
    public int pubVar;
    public void setVar(int priValue,int proValue, int pubValue)
    {
        this.proVar = proValue;
        this.priVar = priValue;
        this.pubVar = pubValue;
    }
    public void getVar()
    {
        System.out.println("Private variable: " + priVar );
        System.out.println("Protected variable: " + proVar );
        System.out.println("Public variable: " + pubVar );
    }
    
};
public class Abstract{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        AccessSpecifierDemo obj = new AccessSpecifierDemo();
        int a,b,c;
        System.out.print("Enter the value of private variable :");
        a = input.nextInt();
        System.out.print("Enter the vaule of protected variable :");
        b = input.nextInt();
        System.out.print("Enter the value of public variable :");
        c = input.nextInt();
        obj.setVar(a,b,c);
        obj.getVar();
        input.close();
    }
}
